### DISCUSSION TITLE
[discussion] tuikit dependecy

### DISCUSSION THREAD

**Discussion by werdahias** [skim-rs/skim#652] [General]
tuikit is the last missing dependecy to get skim into debian (and thus, ubuntu). But it's using an old nix version (0.24). Would be great if you could clarify if 
- tuikit still will be in use in skim in the future
- and if you intend to port it to nix 0.29

**Comment by LoricAndre** [comment]
Hi, we do plan on rewriting skim's UI into ratatui. I wasn't aware that tuikit was an issue in debian, I will try to haste the ratatui rewrite to remove this blocage.

**Reply by werdahias** [reply]
Is the pending MR already something that could be applied as patch ?

**Comment by werdahias** [comment]
great, then I will wait until the ratatui move is completed; all other dependencies are already present in Debian

**Reply by werdahias** [reply]
Debian 13's freeze dates were announced; this means starting on 15th of March, getting packages into the archive will be a higher hurdle (and then impossible).
It'd be great if the ratatui rewrite would be finished mid-February so I can finish the packaging and get it into Debian.

**Reply by LoricAndre** [reply]
I don't think it will be possible for me to be 100% feature-compatible by then unfortunately :/

**Reply by werdahias** [reply]
No worries, once the package has been accepted, I can backport it

**Reply by werdahias** [reply]
actually, soft freeze is the 15th April so if it gets in before that that'd be great. If not I'll backport it

**Comment by werdahias** [comment]
Since it was ported I'll close this; will upload a Debian package soon
